audit-ci.jsonc

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Is this a standard list pulled from the template? Some of these may or may not apply depending on what features we're using in specific repos

Reply:
the first 5 (before the OZ one) are from the template and do affect this repo. the rest were added just for this repo

Reply:
there are 3 signature malleability issues that I held off on whitelisting